src/openforms/authentication/contrib/digid_oidc/middleware.py

```diff
@@ -0,0 +1,13 @@
+from mozilla_django_oidc.middleware import SessionRefresh
+
+from .mixins import SoloConfigMixin
+from .models import OpenIDConnectPublicConfig
+
+
+class SessionRefresh(SoloConfigMixin, SessionRefresh):
```

Comment:
can probably be removed if we remove the custom auth backends from `AUTHENTICATION_BACKENDS` setting?